docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,11 @@
+### `cosmos/cosmos-sdk` Release Process
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a P.R. for the release
+- [ ] 2. Add commits/PRs that are desired for this release **that haven’t already been added to develop**
+- [ ] 3. Ensure that `CHANGELOG.md` contains links to issues/PRs for each item
+- [ ] 4. Summarize breaking API changes section under “Breaking Changes” section to the `CHANGELOG.md` to bring attention to any breaking API changes that affect RPC consumers
+- [ ] 5. Tag the commit `{{ .Release.Name }}-rcN`
+- [ ] 6. Kick off 1 day of automated fuzz testing
+- [ ] 7. Release Lead assigns 2 people to perform buddy testing script
```

Comment:
Should we elaborate more on what this actual testing entails? (6 and 7)

Reply:
I would like to get the [buddy testing script](https://docs.google.com/document/d/1KJJy1Llg83CxtPnNYdz86G67sUqTvyjcHkm7A3CTQ2A/edit) in a place where we can merge it in here and link directly to it.

Reply:
Once we have that finalized, can we put that in the repo somewhere? It'd be nice for everything to be in one place I think.